components/api_request_helper/api_request_helper.cc

```diff
@@ -268,12 +268,20 @@ APIRequestHelper::Ticket APIRequestHelper::CreateURLLoaderHandler(
     request->method = method;
   }
 
+  DVLOG(4) << method << " " << url.spec();
+
   if (!headers.empty()) {
     for (auto entry : headers) {
+      DVLOG(4) << "> " << entry.first << ": " << entry.second;
       request->headers.SetHeader(entry.first, entry.second);
     }
   }
 
+  if (!payload.empty()) {
+    DVLOG(4) << "Payload type " << payload_content_type << ":";
+    DVLOG(4) << payload;
```

Comment:
are we logging the page content on the command line here? Mightn't these contain PII?

Reply:
Perhaps but this is debug builds only and verbose flag level 4. Is that acceptable?